src/benchmark/tpch_benchmark.cpp

```diff
@@ -40,12 +41,16 @@ int main(int argc, char* argv[]) {
 
   // clang-format off
   cli_options.add_options()
-    ("s,scale", "Database scale factor (1.0 ~ 1GB)", cxxopts::value<float>()->default_value("1"))
+    ("s,scale", "Database scale factor (1.0 ~ 1GB)", cxxopts::value<float>()->default_value("10"))
```

Comment:
Change the explanation to `(10.0 ~ 10 GB)`? People might mistake it as the default.